cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/vsphere/vsphere-mc-common/VsphereClusterSettingsStep.tsx

```diff
@@ -77,6 +79,10 @@ export function VsphereClusterSettingsStep(props: Partial<StepProps>) {
         setValue,
     } = methods;
 
+    // update tab status bar
+    if (updateTabStatus) {
```

Comment:
nit: Since UseUpdateTabStatus protects against a null updateTabStatus (and since we expect it to always be non-null), I'm wondering if the extra `if` protection here (and other places) could be removed?

Reply:
@swalner-vmware unfortunately, we can't remove the if condition because we are using Partial for the property. (`Partial<StepProps>`)